### ISSUE TITLE
QOS modification in LDAP fails

### ISSUE THREAD

**Issue by dwgnr** [th-nuernberg/usermgmt#29]
When the QOS of an existing user is modified and one of the QOS values already exists in LDAP, the modification fails with the following error: 

```
[INFO  usermgmt::ldap::ldap] User modification in LDAP failed! LDAP operation result: rc=20 (attributeOrValueExists), dn: "", text: "modify/add: slurmQos: value #0 already exists"
```

We should ignore the existing QOS in order to avoid the error.

**Comment by dwgnr** [comment]
This might be related to #1

**Comment by BoolPurist** [comment]
Was fixed with #42
